### PR TITLE
Livetile updates

### PR DESCRIPTION
**Pull request by ruzzyr**
- Live tiles are flipping.
- Assets are renamed and up to the standard now.
- unity deletes files that are being created and not being used i.e WideTile.png
- in case that stock build project option is used, files that are created by unity ignored.

### CODE REVIEW

File: Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs [hunk 1]
```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
             {
                 throw new Exception(error); // ensures exit code != 0.
             }
+#if UNITY_EDITOR
```
Comment (robertocaldas) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: When is it not going to be UNITY_EDITOR? This is an editor class.

File: Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs [hunk 2]
```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
...
+            RemoveIncorrectAssets(outputPath);
+#endif
         }
+        static void RemoveIncorrectAssets(string outputPath)
+        {
+            string projectName = PlayerSettings.productName;
+            string assetsPath = outputPath + "/" + projectName + "/" + projectName + ".Windows/Assets/";
+
+            //removing unused assets for windows
+            if (File.Exists(assetsPath + "WideTile.scale-100.png"))
```
Comment (robertocaldas) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: MSDN: "If the file to be deleted does not exist, no exception is thrown."
No need to verify if file exists.

File: Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs [hunk 3]
```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
...
+                File.Delete(assetsPath + "WideTile.scale-100.png");
+
+            if (File.Exists(assetsPath + "MediumTile.scale-100.png"))
+                File.Delete(assetsPath + "MediumTile.scale-100.png");
+
+            //removing unused assets for windowsPhone
+            assetsPath = outputPath + "/" + projectName + "/" + projectName + ".WindowsPhone/Assets/";
```
Comment (robertocaldas) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: Please check with Freddy or Keith but I think the Windows folders locations in relation to the Unity Asset folder may vary from project to project (there's a script that's is used to set up a new project, and you define these paths when running it), so it won't work for every project.

Comment (ruarchi) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: outputPath - path of the project
projectName - name of the project
these are coming from the script that you are talking about.

Comment (ruarchi) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: and this is not Unity Asset folder... this is the Asset folder related to the project. And this is the script that sets up the project that you are reviewing.

Comment (keithpatton) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: I think it should be ok, assuming the 100 scale image is the only one generated by unity, so this should work ok

Comment (robertocaldas) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: > And this is the script that sets up the project that you are reviewing.
No, this is the script I wrote that builds the unity project, I was referring to a script that sets up the whole project, it's called `BuildScripts/Build_Local.bat`. But anyway, you are right, the output path parameter is there to do the right thing.

File: Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs (hunk 2 above)
Comment (keithpatton) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: agree with roberto here, you could clean up the code here by just calling File.Delete

File: Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs (hunk 1 above)
Comment (keithpatton) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: @robertocaldas what is your recommendation here for Ruslan, that he remove UNITY_EDITOR or wrap the class in an ifdef. If you can please suggest a solution here that would be great.

Comment (robertocaldas) [inline Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs]: Classes inside the Editor folder are editor only, so no reason to have UNITY_EDITOR in the file.

**Pull request merged**